app/javascript/components/Settings/PaymentsPage/AusBackTaxesSection.tsx

```diff
@@ -2,6 +2,17 @@ import * as React from "react";
 
 import { TaxesCollectionModal } from "$app/components/server-components/TaxesCollectionModal";
 
+export type AusBacktaxDetails = {
+  total_amount_to_au: string;
+  au_backtax_amount: string;
+  credit_creation_date: string;
+  opt_in_date: string | null;
+  opted_in_to_au_backtax: boolean;
+  legal_entity_name: string | null;
```

Comment:
The fix: make `legal_entity_name` nullable. These props were duplicate across 2 components, so I factored them into `AusBacktaxDetails`